docs/source-app/workflows/ssh/index.rst

```diff
@@ -0,0 +1,119 @@
+
+#################################
+Debug cloud apps via SSH
+#################################
+
+**Audience:** Users that want to debug their cloud apps on their own machine
+
+You can
```

Comment:
this needs more context

Reply:
tried to give more context - LMK if you have ideas how to phrase this better.